src/main/java/eu/gir/girsignals/init/GIRSounds.java

```diff
@@ -0,0 +1,35 @@
+package eu.gir.girsignals.init;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import eu.gir.girsignals.GirsignalsMain;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundEvent;
+import net.minecraftforge.event.RegistryEvent;
+
+public class GIRSounds {
+    public static final List<SoundEvent> SOUNDS = new LinkedList<>();
+    
+    public static SoundEvent andreas_cross;
+    
+    public static void register() {
+        andreas_cross = registerSound(GirsignalsMain.MODID + ":andreas_cross");
```

Comment:
Das muss wirlich nicht sein das kann unten automatisiert werden:

```suggestion
        andreas_cross = registerSound("andreas_cross");
```